systest/audit_encrypted/docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+version: "3.5"
+services:
+  alpha1:
+    image: dgraph/dgraph:local
+    working_dir: /data/alpha1
+    ports:
+      - "8080"
+      - "9080"
+    volumes:
+      - type: bind
+        source: $GOPATH/bin
+        target: /gobin
+        read_only: true
+      - type: bind
```

Comment:
use a volume instead

Reply:
> 1. Do we need the .gitkeep files?

I believe so, since we want to keep the audit_dir directory structure intact.  See e.g. systest/audit.  I copied over the directories from there.

Reply:
You can use a volume (not volume mount) and probably you wouldn't need those folders.

Reply:
done

Reply:
moving back to a bind mount - as we discussed using a mount volume created other issues and is not necessary here since we are not persisting data beyond the life of the test.  The only "issue" is that .gitkeep directories are aesthetically unpleasing, but we already do this elsewhere in the repo (e.g. in systest/audit package) so we can revisit this later in the context of making a broader change (to other test packages as well)